diceRoll/diceRollUnitTests.js

```diff
@@ -98,87 +93,87 @@ function testList() {
       app.toggleMenu = new Stub();
       app.userSaveButtonCheckDisplay = new Stub();
       app.run();
-      assert.compare(app.addRollBarListeners.info.called, 1, 'addRollBarListeners() not called');
-      assert.compare(app.addCalculatorListeners.info.called, 1, 'addCalculatorListeners() not called');
-      assert.compare(app.addUserSaveButtonListener.info.called, 1, 'addUserSaveButtonListener() not called');
-      assert.compare(app.simulateFirstVisit.info.called, 1, 'simulateFirstVisit() not called');
-      assert.compare(app.checkMemory.info.called, 1, 'checkMemory() not called');
-      assert.compare(app.toggleMenu.info.called, 1, 'toggleMenu() not called');
-      assert.compare(app.userSaveButtonCheckDisplay.info.called, 1, 'userSaveButtonCheckDisplay() not called');
+      assert.compare(app.addRollBarListeners.info.calls.length, 1, 'addRollBarListeners() not called');
+      assert.compare(app.addCalculatorListeners.info.calls.length, 1, 'addCalculatorListeners() not called');
+      assert.compare(app.addUserSaveButtonListener.info.calls.length, 1, 'addUserSaveButtonListener() not called');
+      assert.compare(app.simulateFirstVisit.info.calls.length, 1, 'simulateFirstVisit() not called');
+      assert.compare(app.checkMemory.info.calls.length, 1, 'checkMemory() not called');
+      assert.compare(app.toggleMenu.info.calls.length, 1, 'toggleMenu() not called');
+      assert.compare(app.userSaveButtonCheckDisplay.info.calls.length, 1, 'userSaveButtonCheckDisplay() not called');
       return true;
     }),
     new UnitTest('addCalculatorListeners()', function(app, test) {
```

Comment:
Name of test: "addCalculatorListeners() calls methods to attach events to the Number, Dice and Operator buttons"